db_upgrade_scripts/mosip_digitalcard/upgrade.sh

```
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+properties_file="$1"
+echo `date "+%m/%d/%Y %H:%M:%S"` ": $properties_file"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)